week1/homework/responses/1.3-first-app-and-events.md

```diff
@@ -0,0 +1,19 @@
+# Your First Slack App
+
+Now that you have done some research on Slack development, its time to get coding! 
+
+## 🛠 Building your App
+
+You will be building your [first app](https://slack.dev/bolt-js/tutorial/getting-started#:~:text=%20Getting%20started%20with%20Bolt%20for%20JavaScript%20,handled%2C%20it%E2%80%99s%20time%20to%20set%20up...%20More%20%E2%80%A9https://slack.dev/bolt-js/tutorial/getting-started), which is a bot that listens for and responds to specific text in chats. **Note:** in order to make a public facing url, you first will need to [install ngrok](https://api.slack.com/tutorials/tunneling-with-ngrok), which dedicates a server on your end to receiving requests from the slack app. 
+
+While you're following both tutorials, write down any questions you have (ie what is this dependency, what is a scope). Then, research one of those questions and submit a 2-4 sentence summary of your findings as a comment. 
+
+After you have completed the project, try changing what the app listens for (ie instead of "hello" it checks for "bit camp"). See if there is anything you can't listen for as part of a chat. 
+
+## 🔔 Learning about Events
+
+A large part of the app you just built is a **slack event**. Understanding events will be important as we will be using them throughout the bootcamp. Skim through the following articles to learn more about what an event is in programming, how events are used with the Slack Events API, and types of events that you can use in a project:
```

Comment:
maybe think about adding in videos/visuals? depending on the student, they might prefer watching something over reading articles